### PR TITLE
Theming: Added the ability for themes to customize WET variables

### PR DESCRIPTION
**Pull request by LaurentGoderre**

### CODE REVIEW

**Comment by LaurentGoderre** [comment]
When this gets merged, all themes will need to be modified accordingly

File: src/_variables.scss [hunk 1]
```diff
@@ -14,115 +10,96 @@ $clrMedium: #ccc;
...
+// Calendar
+$cal-text-head: #fff !default;
+$cal-text-body: #000 !default;
+$cal-dark: #333 !default;
+$cal-medium: #555 !default;
+$cal-active: #ccc !default;
+$cal-days-link: #111 !default;
+
+//Multimedia Player
+$mm-progress-fg-color: #176ca7 !default;
```
Comment (LaurentGoderre) [inline src/_variables.scss]: @melodyma notice that we need to add `!default` for each variable to allow the variable to be customizable by themes

Comment (nschonni) [inline src/_variables.scss]: Not :100: sure that libsass properly supports `!default` right now, but I haven't been following the 3.x dev that close

File: src/base/_bootstrap.scss [hunk 2]
```diff
@@ -0,0 +1,13 @@
+// Hack for Bootstrap 3.1.1, since it doesn't convert the "fadein" LESS method.
+// Pending fix release in https://github.com/twbs/bootstrap-sass/commit/df7be4f0412497e22f1d271d94d6e9835f61113e
+@function fadein($colour, $pct) {
```
Comment (nschonni) [inline src/base/_bootstrap.scss]: This might not be needed anymore since the Bootstap version has been bumped

File: src/_variables.scss (hunk 1 above)
Comment (LaurentGoderre) [inline src/_variables.scss]: It does

File: src/base/_bootstrap.scss (hunk 2 above)
Comment (LaurentGoderre) [inline src/base/_bootstrap.scss]: What version is needed for it to work? Nonetheless, this is something that can be addressed by another PR

Comment (LaurentGoderre) [inline src/base/_bootstrap.scss]: It seems like it's still needed, `_variables.scss` still uses it.... Which is weird because it shouldn't

Comment (LaurentGoderre) [inline src/base/_bootstrap.scss]: Actually no, `fadein` is being converted to `fade_in` so it can go.

**Comment by LaurentGoderre** [comment]
@melodyma you are free to start applying your changes now

**Pull request merged**